internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -67,36 +66,16 @@ var Manifest = &appsv1.DaemonSet{
 				ServiceAccountName: config.OperatorName,
 				InitContainers: []v1.Container{
 					{
-						Name: "non-root-enabler",
-						// Creates directory /var/lib/security-profiles-operator,
-						// sets 65535:65535 as its owner and symlink it to
-						// /var/lib/kubelet/seccomp/operator. This is required
-						// to allow the main container to run as non-root.
-						Command: []string{"bash", "-c"},
-						Args: []string{`
-						set -euo pipefail
-
-						if [ ! -d ` + config.KubeletSeccompRootPath + ` ]; then
-							/bin/mkdir -m 0744 -p ` + config.KubeletSeccompRootPath + `
-						fi
-
-						/bin/mkdir -p ` + operatorRoot + `
-						/bin/chmod 0744 ` + operatorRoot + `
-
-						if [ ! -L ` + config.ProfilesRootPath + ` ]; then
-							/bin/ln -s ` + operatorRoot + ` ` + config.ProfilesRootPath + `
-						fi
-
-						/bin/chown -R 65535:65535 ` + operatorRoot + `
-						cp -f -v /opt/seccomp-profiles/* ` + config.KubeletSeccompRootPath + `
-					`},
+						Name:            "non-root-enabler",
+						Args:            []string{"non-root-enabler"},
+						ImagePullPolicy: v1.PullNever,
```

Comment:
The daemon container defaults to `PullAlways`. Let's make both containers default to the same pull policy instead.

Reply:
Sure, fixing as suggested :+1: